.github/workflows/deploy.yml

```diff
@@ -29,6 +29,7 @@ jobs:
             --canary \
             --dist-tag=next \
             --exact \
+            --preid=$(git rev-parse --short "$GITHUB_SHA") \
```

Comment:
Why use `$GITHUB_SHA` rather than `HEAD`?

Reply:
What is `HEAD`? It's not listed in https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
It's the "head" of the current branch.